examples/bootstrapping/vagrant/onboarding/README.md

```diff
@@ -0,0 +1,117 @@
+# Onboarding 
+
+`NOTE: This is not a recommended architecture but rather meant to help you get started on your journey with Chef.`
+
+The purpose of this project is to have you become comfortable with leveraging the Chef toolkit to identify and remediate a compliance issue on a node. During this exercise you will provision, configure and use components like Chef Automate, Chef Infra Server, Chef Inspec and Policyfiles. If you find something you think should be changed, updated, added or removed please submit a Pull Request (PR). Enjoy!
```

Comment:
You should add a contributing section (at the bottom) and put the last sentence in this section there.  I'd also remove the word _like_ in your description here.  Let's try to be as explicit as possible.  Be very deliberate as to what this does, for example: 

This will create X vagrant boxes, one chef server, one chef automate server and a chef node.  The purpose of spinning up multiple vagrant boxes is to create an environment where one can verify bootstrapping of nodes or become more familiar with the bootstrap process. InSpec will be run against the node after the bootstrapping process to demonstrate X

Reply:
Updated.